test/index-test.js

```diff
@@ -146,7 +150,10 @@ module.exports = (repo) => {
       })
     })
 
-    it('block is retrived from peer', (done) => {
+    // Not sure if I understand what is going on here
+    // test fails because now the network is not properly mocked
+    // what are these net.stores and mockNet.bitswaps?
```

Comment:
Yes, I need to change the mock network tool to now mock libp2p instead of the network instance